src/modules/blocks/event-datetime/controls/template.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	PanelBody,
+	SelectControl,
+	TextControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import {
+	date,
+} from '@moderntribe/common/utils';
+
+/**
+ * Module Code
+ */
+
+const { timezonesAsSelectData } = date;
+
+class EventDateTimeControls extends PureComponent {
+	static propTypes = {
+		isEditable: PropTypes.bool.isRequired,
+		onTimeZoneVisibilityChange: PropTypes.func,
+		separatorDate: PropTypes.string,
+		separatorTime: PropTypes.string,
+		setSeparatorDate: PropTypes.func,
+		setSeparatorTime: PropTypes.func,
+		setTimeZone: PropTypes.func,
+		showTimeZone: PropTypes.bool,
+		timeZone: PropTypes.string,
+	};
+
+	/**
+	 * Controls being rendered on the sidebar.
+	 *
+	 * @returns {ReactDOM} A React Dom Element null if none.
+	 */
+	renderControls = () => {
+		const {
+			onTimeZoneVisibilityChange,
+			separatorDate,
+			separatorTime,
+			setSeparatorDate,
+			setSeparatorTime,
+			setTimeZone,
+			showTimeZone,
+			timeZone,
+		} = this.props;
+
+		// @todo: modify so this code does not fire unless the block is selected
```

Comment:
do we still need this todo here?

Reply:
I actually have no idea on that, I just left it because it was already there. 

@mitogh is it save to delete this comment? I'm not sure what it was intended for

Reply:
Someone can correct me if I'm wrong, but I think in an old implementation of gutenberg the inspector controls would fire even if the block wasn't selected (but now it's fixed on gutenberg's side) and it may have been for that?

Reply:
I think that's right we might be able to get ride of that `TODO`

Reply:
Consider it :boom: